baselines/bigquery-storage/src/v1beta1/big_query_storage_client.ts.baseline

```diff
@@ -186,6 +186,7 @@ export class BigQueryStorageClient {
         this._opts.fallback ?
           (this._protos as protobuf.Root).lookupService('google.cloud.bigquery.storage.v1beta1.BigQueryStorage') :
           // tslint:disable-next-line no-any
```

Comment:
You can delete the `tslint` specific comments here

Reply:
removed tslint no-any comment.